ios/COVIDSafePaths.xcodeproj/project.pbxproj

```diff
@@ -708,11 +702,11 @@
 			children = (
 				B2A58213244CE76C000F39A6 /* en */,
 				B2A58216244CE778000F39A6 /* it */,
-				B2A5821E244CE988000F39A6 /* fr */,
+				B2A5821E244CE988000F39A6 /* sk */,
 				B2A58220244CE9EB000F39A6 /* ro */,
 				B2A58222244CE9F7000F39A6 /* id */,
 				B2A58224244CEA20000F39A6 /* fr-HT */,
-				B2A58226244CEA6C000F39A6 /* ml */,
+				B2A58226244CEA6C000F39A6 /* pt-BR */,
```

Comment:
We lost/changed a couple of languages. 

fr is gone, ml is gone.

Can you revert the language related changes in this file?

Reply:
@tstirrat As we've discussed in Slack, these changes were automatically completed by Xcode due to those string translations being empty files. I'd hesitate to modify this file directly going against Xcode's auto-modify. I'm not 100% sure what we are missing by not having these translations listed here. The fallback would always be to show in a default language. I don't think I've got time to look much further into it. Can you or someone else take a further look?

Reply:
We can take that offline. As long as ft-HT is still there its not blocking prod builds.